manifests/mark.pp

```diff
@@ -30,7 +30,7 @@
   $onlyif_cmd = [['/usr/bin/dpkg', '-l', $title]]
   $command = ['/usr/bin/apt-mark', $setting, $title]
 
-  exec { "apt-mark ${setting} ${title}":
+  exec { ['apt-mark', $setting, $title]:
```

Comment:
This isn't the actual command, so shouldn't be changed.

Reply:
@kenyon I was testing as the new version of puppet-lint reporting for title as well, have created ticket for Team for followup on the same.

Reply:
BTW Thanks for your review.